controllers/mapController.js

```diff
@@ -21,4 +23,29 @@ const getMapPoints = async (req, res, next) => {
   }
 };
 
+const createNewMap = async (req, res, next) => {
+  const { map, user } = req.body;
+
+  try {
+    const currentUser = await User.findOne({ _id: user }).exec();
+    const newMap = new Map(map);
+
+    newMap.members.push(currentUser);
+    currentUser.myMaps.push(newMap);
+
+    await newMap.save();
+    await currentUser.save();
```

Comment:
두 실행문이 서로 연관되어 있지 않으니 병렬적으로 실행시키면 조금 더 빠르게 클라이언트 쪽에 응답을 내어줄 수 있을 것 같은데 어떻게 생각하실 까요?

```suggestion
    await Promise.all([newMap.save(), currentUser.save()]);
```